src/e2e-test/java/io/cdap/plugin/pubsub/stepsdesign/PubSubSink.java

```diff
@@ -234,4 +237,33 @@ public void enterRuntimeArgumentValueForPubSubSinkPropertyTopicKey(String runtim
   public void clickOnPreviewDataForPubSubSink() {
     openSinkPluginPreviewData("GooglePublisher");
   }
+
+  @Then("Subscribe to the messages")
```

Comment:
Subscribe to the Pub/Sub messages seems more detailed and informative

Reply:
done.